uint8 MAIN_ALLY=0
uint8 SECONDARY_ALLY=1
uint8 MAIN_OPPONENT=2
uint8 SECONDARY_OPPONENT=3



uint8 robot_id
geometry_msgs/PoseWithCovarianceStamped pose
geometry_msgs/TwistWithCovarianceStamped twist